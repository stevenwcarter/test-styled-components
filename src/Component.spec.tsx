import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow } from "enzyme";
import "jest-styled-components";
import TestComponent from "./Component";

configure({ adapter: new Adapter() });

describe("Test", () => {
  it("should match snapshot", () => {
    const instance = shallow(<TestComponent />);
    expect(shallow(<TestComponent />)).toMatchSnapshot();
  });
});
