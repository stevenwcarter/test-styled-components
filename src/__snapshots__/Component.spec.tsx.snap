// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test should match snapshot 1`] = `
<Component__StyledWrapper>
  <Component__StyledDiv>
    TEST
  </Component__StyledDiv>
</Component__StyledWrapper>
`;
